src/templates.c

```diff
@@ -501,6 +524,18 @@ gchar *templates_get_template_fileheader(gint filetype_idx, const gchar *fname)
 	return g_string_free(template, FALSE);
 }
 
+// The line `GEANY_API_SYMBOL` is needeed ?
```

Comment:
The function templates_get_template_fileheader had a line with GEANY_API_SYMBOL just before the function definition, I don't know exactly what it does, but it maybe needed also for the function below.

Reply:
Yes, it would be necessary here as well. This helps the linker to make the function available for the plugin API.

BUT: I think we don't need the "fileheader3" template at all. It doesn't add much value as it is the same as "fileheader" but only for GPLv3.
Since the templates are fully user customizable, it is probably enough for users to modify the existing "fileheader" template or add a new one themselves.

So I suggest to remove the "fileheader3" template altogether.